tests/hub/hub_suite_test.go

```diff
@@ -148,6 +148,7 @@ var _ = BeforeSuite(func() {
 		testSliceEventRecorder,
 		mf,
 	)
+	sr.ReconcileInterval = 5 * time.Second
```

Comment:
Can you please add IT for the same start the UT for the same

Reply:
Added, please review again